src/pages/patientView/timeline2/ClinicalEventsTables.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { ClinicalEvent } from 'cbioportal-ts-api-client';
+import { groupTimelineData } from 'pages/patientView/timeline2/timeline-data-utils';
```

Comment:
Not sure if we should use `-` in filenames. I would name this file `TimelineDataUtils` or something like that.